rules-testing/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "rules-testing",
+  "version": "1.0.0",
+  "description": "Unit testing for firestore rules",
+  "main": "test.js",
+  "scripts": {
+    "test": "mocha --exit"
+  },
+  "author": "",
+  "license": "APACHE 2.0",
+  "devDependencies": {
+    "@firebase/testing": "^0.20.9",
+    "mocha": "^8.0.1"
```

Comment:
So I know you said you accidentally installed mocha so is it supposed to be listed here?

Reply:
Actually yes! That is why I got confused. The firestore rules testing use mocha. Jasmine and mocha are very similar, and it was the easiest way I found on setting up the testing framework with the rules.